docs/team/weijie96.md

```diff
@@ -12,26 +12,45 @@ It is written in Java, and has about 10 kLoC.
+  * Highlights: 
+  This enhancement affects existing commands (such as `ListCommand` and `AddCommand`), and commands to be added in future (such as `FindCommand`). 
+  It required an in-depth analysis of design alternatives. 
+  The implementation too was challenging as it required changes to existing commands.
+  The feature is complete, as both archiving and unarchiving are implemented, in addition to viewing the archive.
   
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=WeiJie96&tabRepo=AY2021S1-CS2103-T16-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other)
 
 * **Enhancements to existing features**:
-    * **Coming Soon**
-
+  * Made fields in `Person`, such as `Email` and `Address`, optional.
+  (Pull requests [\#41](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/41), [\#60](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/60))
+  
 * **Documentation**:
   * User Guide:
-    * Added documentation for the following features: **Coming Soon**
-    * Did cosmetic tweaks to existing documentation of the following features: **Coming Soon**
-    * **Coming Soon**
+    * Added/Maintained documentation for `add`, `archive`, `clear`, `exit`,  `help`, `list`, `list r/` and `unarchive` commands.
+      (Pull requests [\#68](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/68),
+      [\#134](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/134), [\#138](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/138))
+    * Added an individual section to explain the `archive` feature. [\#159](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/159)
+    * Maintained FAQ and Quick Start. [\#159](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/159)
   * Developer Guide:
     * Added implementation details of the `archive` feature.
-    * **Coming Soon**
+    (Pull requests [\#83](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/83), [\#131](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/131))
+    * Added/Maintained user stories, use cases and test cases for `archive`, `delete` `help`, `list`, `list r/` and `unarchive` commands. 
+    (Pull requests [\#23](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/23), 
+    [\#69](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/69), [\#160](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/160))
 
+* **Contributions to team-based tasks**:
+  * PRs reviewed (examples with non-trivial review comments): 
+  [\#50](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/50), [\#97](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/97), [\#124](https://github.com/AY2021S1-CS2103-T16-2/tp/pull/124)
+  * Maintained the issue tracker.
+  
```

Comment:
I would say you are a really solid code quality checker. Anyways I would suggest like we meet on Monday and go through our good points (or bad) in person, since it is much easier to like see others pros and cons instead of writing it yourself.

Reply:
Added a short point for this 👍